go-controller/pkg/libovsdbops/db_object_types.go

```diff
@@ -80,3 +83,16 @@ var AddressSetEgressService = newObjectIDsType(addressSet, EgressServiceOwnerTyp
 	ObjectNameKey,
 	AddressSetIPFamilyKey,
 })
+
+var ACLMulticastNamespace = newObjectIDsType(acl, MulticastNamespaceOwnerType, []ExternalIDKey{
+	// namespace
+	ObjectNameKey,
+	// there are 2 possible values for namespaced-acls: allow+ingress/egress
+	TypeKey,
```

Comment:
I was wondering this too when I saw priority is stored as an external id as well as stored in priority column of acl.

Reply:
With AddressSet we really did not have any other column to pull IDs from. All its IDs are concepts ovn-k handles but not reflected in the table itself. So there we did not have any other option than having them as external IDs.

But this is not the same situation for other tables. Some tables might have a better place to hold the primary ID, like for example LB name column, etc. Other tables might already have columns that hold secondary IDs like the ACLs here.

I do find a case for having a primary ID as external ID everywhere, for commonality, ease of troubleshooting, ease of processing. But even this I find debatable.

Replicating every secondary ID on its own external ID when there is already a column for it is taking it one step further.

So there is also a case of implementing how the DBObjectIDs project onto the tables they refer to.

While I wouldn't worry too much now, the concern I do have is if this becomes a problem, how do we get out of it? Another sync? It might be worth considering now.

Reply:
good points @jcaamano , wdyt  @npinaeva ?

Reply:
We've talked with @jcaamano , and agreed that the ids should be more semantics than implementation based.
So I am updating the IDs to talk more in terms of the k8s objects, in that case we can leave the ids even if we decide to change ACLs that represent them at some point.